windows-gaming-driver/src/driver/mod.rs

```diff
@@ -96,10 +97,22 @@ pub fn run(cfg: &Config, tmp: &Path, data: &Path) {
         Box::new(catch_sigterm),
     ]).map(|_| ());
 
-    match core.run(qemu.select(joined)) {
-        Ok(((), _)) => (), // one (*hopefully* qemu) is done, so the other is too
-        Err((e, _)) => panic!("Unexpected error: {}", e),
-    }
-
+    core.run(qemu.join(joined.or_else(|x| { error!("Unexpected error: {}", x); Ok(()) }))).expect("Unexpected error");
+
+	info!("unbinding resetable vfio-things");
+	
+    for slot in cfg.machine.vfio_slots.iter() {
+		if let &VfioDevice::Temporarily(ref device) = slot {
+			let mut child = Command::new(Path::new(::DATA_FOLDER).join("vfio-ubind")).arg(device).arg("-r").spawn().expect("failed to run vfio-ubind");
```

Comment:
This function's `data` parameter is literally `Path::new(DATA_FOLDER)` (passed through from `main()`).